torch_xla/csrc/ops/bitwise_ir_ops.cpp

```diff
@@ -15,7 +15,7 @@ Value BitwiseAnd(const Value& node1, const Value& node2) {
     auto kernel = [](const xla::XlaOp& op0, const xla::XlaOp& op1) {
       return op0 & op1;
     };
-    return node.ReturnOp(XlaHelpers::PromotedBinaryOp(op0, op1, kernel), loctx);
+    return node.ReturnOp(kernel(op0, op1), loctx);
```

Comment:
What are the rules for bitops promotions in pytorch. Because this PR will end up doing absolutely no promotion.
Is 'int' BITOP 'long' legal?

Reply:
I'll double check and add a few more tests to cover different behaviors.

Reply:
fyi confirmed `int` BITOP `long` is illegal, for BITOP it requires to be the same dtype in pytorch. 
And for bool, it's treated as a separate category so it doesn't allow automatic type promotion to other dtypes. https://github.com/nairbv/pytorch/blob/promote_types_strict/test/test_type_promotion.py#L37